src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  width: $card-width;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    margin: 0 0 4px;
+
+    &--text_medium {
+      @extend %custom-font;
+      font-size: $font-size-medium;
+      font-weight: $font-weight-medium;
+      line-height: $line-height-text-medium;
+      color: $color-text-primary;
+    }
```

Comment:
Why do you create modifier for these styles?
Just put those under element class, you don't have huge variety here.

Reply:
Done.